Chicken_Ingredient.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Cuisines</title>
+  </head>
+  <body>
+    <h1>Chicken Dishes</h1>
+    <ul>
+      <li>
+        <a href="Chicken_Stir_Fry.html"><h2>Chicken Stir Fry Noodles!</h2></a>
```

Comment:
usually with the `a` tag you would nest it inside the `h2` (e.g. `<h2><a href="..">Chicken Stir Fry Noodles</a></h2>`